### PR TITLE
zh: resync concepts workloads files

### PR DESCRIPTION
**Pull request by chenrui333**
part of #27672

/cc @tengqm

### CODE REVIEW

File: content/zh/docs/concepts/workloads/pods/disruptions.md [hunk 1]
```diff
@@ -155,18 +155,23 @@ and [stateful](/docs/tasks/run-application/run-replicated-stateful-application/)
...
-自愿干扰的频率各不相同。在一个基本的 Kubernetes 集群中，根本没有自愿干扰。然而，集群管理
-或托管提供商可能运行一些可能导致自愿干扰的额外服务。例如，节点软
+自愿干扰的频率各不相同。在一个基本的 Kubernetes 集群中，没有自愿干扰（只有用户触发的干扰）。
+然而，集群管理员或托管提供商可能运行一些可能导致自愿干扰的额外服务。例如，节点软
 更新可能导致自愿干扰。另外，集群（节点）自动缩放的某些
 实现可能导致碎片整理和紧缩节点的自愿干扰。集群
 管理员或托管提供商应该已经记录了各级别的自愿干扰（如果有的话）。
+有些配置选项，例如在 pod spec 中
+[使用 PriorityClasses]（https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/）
+也会产生自愿（和非自愿）的干扰
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/disruptions.md:174]:
```suggestion
也会产生自愿（和非自愿）的干扰。
```

File: content/zh/docs/concepts/workloads/pods/init-containers.md [hunk 2]
```diff
@@ -391,7 +391,7 @@ myapp-pod   1/1       Running   0          9m
 
 <!--
 This simple example should provide some inspiration for you to create your own
-init containers. [What's next](#whats-next) contains a link to a more detailed example.
+init containers. [What's next](#what-s-next) contains a link to a more detailed example.
 -->
 这个简单例子应该能为你创建自己的 Init 容器提供一些启发。
 [接下来](#whats-next)节提供了更详细例子的链接。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/init-containers.md:397]:
```suggestion
[接下来](#what-s-next)节提供了更详细例子的链接。
```

File: content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ topology spread constraints.
 -->
 
 {{< note >}}
-在 v1.19 之前的 Kubernetes 版本中，如果要使用 Pod 拓扑扩展约束，你必须在 [API 服务器](/zh/docs/concepts/overview/components/#kube-apiserver) 
+在 v1.19 之前的 Kubernetes 版本中，如果要使用 Pod 拓扑扩展约束，你必须在 [API 服务器](/zh/docs/concepts/overview/components/#kube-apiserver)
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:39]:
```suggestion
在 v1.18 之前的 Kubernetes 版本中，如果要使用 Pod 拓扑扩展约束，你必须在
[API 服务器](/zh/docs/concepts/overview/components/#kube-apiserver)
```

**Comment by tengqm** [comment]
/approve
pls squash your commits into one.

**Review by tanjunchen — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**